src/plugins/interactive_setup/server/plugin.ts

```diff
@@ -13,11 +13,18 @@ import type { CorePreboot, Logger, PluginInitializerContext, PrebootPlugin } fro
 
 import { ElasticsearchConnectionStatus } from '../common';
```

Comment:
__note:__ didn't write tests for this class yet as I think it's too early, let's stabilize implementation a bit more.